.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  useTabs: false,
+  singleQuote: true,
+  semi: false,
+  trailingComma: "none",
+  printWidth: 100
+}
```

Comment:
need trailing newline

Reply:
Will be resolved in the PR submitted for unit tests, post-resolving conflicts.